packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -309,6 +310,12 @@ const ReactTestUtils = {
    * @return {object} the ReactTestUtils object (for chaining)
    */
   mockComponent: function(module, mockTagName) {
+    lowPriorityWarning(
+      false,
+      'ReactTestUtils.mockComponent() is deprecated. ' +
+        'We recommend using shallow rendering or jest.mock() instead.',
+    );
```

Comment:
Minor, but other warnings use more direct language like:

> "ReactTestUtils.mockComponent() is deprecated. Use shallow rendering or jest.mock() instead."`.

This isn't a hard recommendation, so I think softening it a bit makes sense, but it just crossed my mind.

Reply:
Cool, I'm happy with tweaking the wording 😄